CMakeLists.txt

```diff
@@ -71,6 +77,13 @@ endif ()
 get_filename_component(TOKENIZERS_CPP_ROOT ${CMAKE_CURRENT_LIST_FILE} DIRECTORY)
 set(TOKENIZERS_CPP_CARGO_SOURCE_PATH ${TOKENIZERS_CPP_ROOT}/rust)
 
+FetchContent_Declare(
```

Comment:
I wonder the differences between `fetch_content` and setting it as a 3rdparty in the repo

Reply:
There shouldn't be much difference, whether it's 3rd or FetchContent, both pull repositories online for building. Personally, I prefer this approach.